### DISCUSSION TITLE
PDF Reporting improvement

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15861] [Feature improvement]
Following my previous question #15860
As reporting is a bit limited, it would be productive to include (out of the box) text parser function allowing for listing many records in one PDF (email template) document with line items AND summary as per attached sample.  Of course it could be extended to fit various models. Best!
![Custom Advanced Module](https://user-images.githubusercontent.com/84110093/161676432-835c1679-6839-42ad-82e6-6195a0f8c755.jpg)

**Comment by mariuszkrzaczkowski** [comment]
I agree, but we are waiting for an order from someone
